frontend/src/bundles/common/components/select/styles.module.scss

```diff
@@ -22,9 +22,14 @@
 }
 
 .placeholder {
+    padding: 0;
     color: var(--nepal);
 }
 
+.container .input > div {
```

Comment:
We can`t use global tags
B10
https://github.com/BinaryStudioAcademy/quality-criteria/blob/production/src/javascript.md

Reply:
fixed